falcon/util/misc.py

```diff
@@ -145,7 +145,7 @@ def http_date_to_dt(http_date, obs_date=False):
     raise ValueError('time data %r does not match known formats' % http_date)
 
 
-def to_query_str(params):
+def to_query_str(params, encode_lists_as_comma_delimited_string=True):
```

Comment:
I could change it to just `encode_lists` or `comma_delimited_lists`, tell me which you  prefer.